src/typed-method-types/apps.ts

```diff
@@ -183,6 +186,31 @@ type AppsAuthExternalGet = {
   (args: AppsAuthExternalGetArgs): Promise<AppsAuthExternalGetResponse>;
 };
 
+// AppsAuthExternalDelete
+
+type AppsAuthExternalDeleteArgs = {
+  /** @description The id of a specified external token */
+  external_token_id: string;
```

Comment:
Based on the [(hidden) docs](https://api.slack.com/methods/apps.auth.external.delete), it seems like a) there are more parameters potentially to expose and b) they are all optional.

Should this be optional, too, then?

Reply:
Good point @filmaj.
Although there are multiple ways (parameters) to delete the token, as far as I understand, the external developers should only use the `external_token_id` field and not the other two.
This is the reason why I only referenced this one here.
However, let me check with my team again to see if they feel otherwise for some reason. Thanks!

Reply:
circled back with the team. This is what we would want for now (only expose `external_token_id` as a parameter):
https://slack-pde.slack.com/archives/C02TBKP7CAZ/p1675719023065049